### DISCUSSION TITLE
[Scam Alert] Project ( HoHo )

### DISCUSSION THREAD

**Discussion by Blockunity** [Blockunity/WarningList#39] [General]
آدرس سایت 
https://hoho.mobi
ادرس کانترکت
0x3Aabcabf828e684b3D9329831453aE180bD9D2a7 - BSC

https://bscscan.com/address/0x3Aabcabf828e684b3D9329831453aE180bD9D2a7

ادعای این پروژه 
راه اندازی و تبدیل شدن به پلتفرم بازی های بلاکچینی
(مشاهده شده در شهرستان کاشان در حال رشد هستند)

مشکلات و عیب های شناسایی شده

شبکه اجتماعی مانند (تلگرام،اینستاگرام،توییتر،دیسکورد) ندارد و در سایت اشاره ای بهش نشده است . 
فقط در سایت یک قسمت برای چت تهیه شده است که چت ها باید توسط ادمین ها تایید شود .

هویت گردانندگان و تهیه کنندگان پروژه نا مشخص و مخفی است 

نود و نه (99)درصد توکن ها در یک ادرس قرار دارد که بزرگترین هشدار است . 
تعداد کل توکن ها 60,000,000,000 HOHO است که تقریبا اکثرش ماین شده و در داخل یک ولت است و در کل 161 هولدر دارد .

توکن hoho در تاریخ 23 مارچ 2022 راه اندازی شده است و خیلی زمان زیادی نیست.

وایت پیپر پروژه خیلی سطحی نوشته شده است و به هیچ عنوان از نظر فنی صحبتی از کارکرد پروژه نکرده و فقط از دید و اینده ای که دارند صحبت شده است و اکثر صحبت ها از اطلاعات عمومی بلاکچین است .

در کوین مارکت کپ یا کوین گکو لیست نشده است تا اطلاعات بیشتری قابل دریافت باشد .

پروژه گیت هاب ندارد تا کارکرد دولوپر ها قابل مشاهده باشد .

خرید توکن داخل خود سایت و با bnb انجام میشود .

سایت پروژه در تاریخ 2-3-2022 راه اندازی شده است .

اطلاعات whois دامنه سایت مخفی شده است و قابل مشاهده نیست .

توکنومیک و توکن اکونومیک ندارد.

![Screen Shot 2022-07-21 at 2 19 28 PM](https://user-images.githubusercontent.com/81887343/180185817-2cfa2492-4e6f-4ff2-b180-6b8dc8da9cc0.jpg)
![Screen Shot 2022-07-21 at 2 19 35 PM](https://user-images.githubusercontent.com/81887343/180185957-3b01fc70-cf11-40d8-9fd1-bc3fe5130a98.jpg)
![Screen Shot 2022-07-21 at 2 19 42 PM](https://user-images.githubusercontent.com/81887343/180186129-602da997-2a14-48e7-9de3-96c01b305838.jpg)
![Screen Shot 2022-07-21 at 2 19 49 PM](https://user-images.githubusercontent.com/81887343/180186140-1b580a6b-3332-49b5-95ae-d1a316070b25.jpg)
![Screen Shot 2022-07-21 at 2 19 56 PM](https://user-images.githubusercontent.com/81887343/180186155-7e460d19-3345-4e49-9849-bf5b5a8a4621.jpg)
![Screen Shot 2022-07-21 at 2 20 02 PM](https://user-images.githubusercontent.com/81887343/180186167-c7df6a88-c849-4ac6-860d-a9e95e1a2237.jpg)
![Screen Shot 2022-07-21 at 2 20 07 PM](https://user-images.githubusercontent.com/81887343/180186234-620bb717-5076-497f-9488-6681ec53270b.jpg)

**Comment by micro73** [comment]
قبلا بررسي شده بود اين توكن
https://github.com/Blockunity/Warning-List/discussions/9
